packages/ckeditor5-html-embed/docs/features/html-embed.md

```diff
@@ -68,61 +68,35 @@ ClassicEditor
 
 The feature is by default configured to not show previews of the HTML snippets. You can enable the previews by setting the {@link module:html-embed/htmlembedconfig~HtmlEmbedConfig#showPreviews `config.htmlEmbed.showPreviews`} option to `true`.
 
-However, by showing previews of the embedded HTML snippets, you expose the users of your system to the **risk of executing malicious JavaScript code inside the editor**. Therefore, it is highly recommended to plug in some HTML sanitizer that will strip the malicious code from the created snippets before rendering their previews. You can plug in the sanitizer by defining the {@link module:html-embed/htmlembedconfig~HtmlEmbedConfig#sanitizeHtml `config.htmlEmbed.sanitizeHtml`} option.
+However, by showing previews of the embedded HTML snippets, you expose the users of your system to the **risk of executing malicious JavaScript code inside the editor**. Therefore, it is highly recommended to plug in some HTML sanitizer that will strip the malicious code from the created snippets before rendering their previews. You can plug in the sanitizer by defining the {@link module:core/editor/editorconfig~EditorConfig#sanitizeHtml `config.sanitizeHtml`} option.
 
 ```js
 ClassicEditor
 	.create( document.querySelector( '#editor' ), {
 		plugins: [ HtmlEmbed, /* ... */ ],
 		toolbar: [ 'htmlEmbed', /* ... */ ],
 		htmlEmbed: {
-			showPreviews: true,
-			sanitizeHtml: ( inputHtml ) => {
-				// Strip unsafe elements and attributes, for example:
-				// the `<script>` elements and `on*` attributes.
-				const outputHtml = sanitize( inputHtml );
-
-				return {
-					html: outputHtml,
-					// true or false depending on whether the sanitizer stripped anything.
-					hasChanged: true
-				};
-			}
+			showPreviews: true
+		},
+		sanitizeHtml: ( inputHtml ) => {
+			// Strip unsafe elements and attributes, for example:
+			// the `<script>` elements and `on*` attributes.
+			const outputHtml = sanitize( inputHtml );
+
+			return {
+				html: outputHtml,
+				// true or false depending on whether the sanitizer stripped anything.
+				hasChanged: true
+			};
 		}
 	} )
 	.then( /* ... */ )
 	.catch( /* ... */ );
 ```
 
-Currently, the [feature does not execute `<script>` tags](https://github.com/ckeditor/ckeditor5/issues/8326) so the content that requires executing JavaScript to generate a preview will not show in the editor. However, other JavaScript code, for example, used in `on*` observers and `src="javascript:..."` attributes will be executed. You still need to enable the sanitizer.
+### Content Security Policy
```

Comment:
I am missing a reference to the new guide ("HTML security") somewhere here, probably just after the snippet above.